pricing-page-exercise/mesut/style.css

```diff
@@ -0,0 +1,342 @@
+* {
+    box-sizing: border-box;
+    font-family: "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+}
+
+body {
+    margin: 0;
+    
+}
+
+
+.navbar {
+    height: 70px;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    border-bottom: solid 1px #dfdfdf;
+    padding: 16px 24px;
+    margin-bottom: 16px;
+    box-shadow: 0px 2px 10px #a2a0a04c;
+}
+
+.top-right-links {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+
+}
+
+.top-right-buttons {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    color: #1D2124;
+    font-size: 16px;
+}
+
+.top-right-list {
+    padding: 12px;
+    color: #1d2124;
+
+}
+
+.company-link {
+    padding: 10px;
+    font-size: 20px;
+    color: #212529;
+    white-space: nowrap;
+    justify-content: start;
+}
+
+
+.link {
+    text-decoration: none;
+}
+
+
+.top-right-buttons:visited {
+    color: #1D2124;
+}
+
+.features-button:hover {
+    text-decoration: underline;
+}
+
+.enterprise-button:hover {
+    text-decoration: underline;
+}
+
+.support-button:hover {
+    text-decoration: underline;
+}
+
+.pricing-button:hover {
+    text-decoration: underline;
+}
+
+
+.signup-button {
+    color: #007BFF !important;
+    padding: 7px 12px;
+    border: 1px solid #007BFF;
+    border-radius: 5px;
+    white-space: nowrap;
+}
+
+.signup-button:hover {
+    color: #ffffff !important;
+    background-color: #007BFF;
+    transition: 0.3s;
+}
+
+.container {
+    max-width: 975px;
+    justify-content: center;
+    margin: 0 auto;
+}
+
+
+.pricing {
+
+    margin: 0 auto;
+    padding: 8px 0;
+    max-width: 800px;
+}
+
+.pricing-header {
+    text-align: center;
+    font-size: 56px;
+    font-weight: 200;
+    margin-bottom: 0;
+}
+
+.pricing-text {
+    text-align: center;
+    font-size: 20px;
+    font-weight: 100;
+    line-height: 30px;
+    padding: 0 50px;
+}
+
+.cards {
+    display: flex;
+    justify-content: center;
+}
+
+.card {
+    border: 1px solid #DFDFDF;
+    text-align: center;
+    width: 33%;
+    margin: 20px;
+    border-radius: 5px;
+    box-shadow: 0 1px 5px #00000013;
+}
+
+.card-list {
+    padding: 0;
+}
+
+.subscription-type {
+    width: 100%;
+    border-bottom: 1px solid #DFDFDF;
+    margin: 0;
+    padding: 10px 0;
+    background-color: #F7F7F7;
+    font-weight: 400;
+    border-radius: 5px;
+}
+
+.card-text {
+    list-style-type: none;
+    padding: 0 20px;
+    margin: 0;
+    font-size: 16px;
+    line-height: 24px;
+    white-space: nowrap;
+
+}
+
+.price-and-text {
+
+    padding: 20px;
+
+}
+
+.dollar {
+    font-size: 40px;
+    font-weight: 500;
+}
+
+.month {
+    font-size: 32px;
+    color: #6c757d;
+    font-weight: 400;
+}
+
+.price-button-white {
```

Comment:
There's a lot of code duplcation between `.price-button-white` and `.price-button-blue`.
Try using a class like `.price-button` to add the styles that are the same, and then use `.price-button-white` and `.price-button-blue` to only add styles that are different.